src/main/java/org/broadinstitute/hellbender/tools/ApplyBQSRWithoutMinQScoreArgumentCollection.java

```diff
@@ -0,0 +1,51 @@
+package org.broadinstitute.hellbender.tools;
+
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.ArgumentCollectionDefinition;
+import org.broadinstitute.hellbender.utils.QualityUtils;
+
+import java.io.Serializable;
+
+public class ApplyBQSRWithoutMinQScoreArgumentCollection implements ArgumentCollectionDefinition, Serializable {
```

Comment:
Makes sense, combining multiple tools' arguments is stretching the argument parser farther than it was initially intended. 

Is it possible to refactor so that this code isn't replicated in both places?  If that's not doable could you extract the common argument names into shared constants so that they stay in sync?